.github/workflows/cypress.yml

```diff
@@ -30,8 +30,8 @@ jobs:
         uses: skjnldsv/read-package-engines-version-actions@v1.2
         id: versions
         with:
-          fallbackNode: '^12'
-          fallbackNpm: '^6'
+          fallbackNode: '^16'
+          fallbackNpm: '^7'
```

Comment:
Not really necessary :wink:

Reply:
I think the workflows are updated from the organization repo anyways, so they should be updated there to fallback to the latest supported version.
I will undo that change :)